### PR TITLE
merge drawing steps into one loop

### PR DESCRIPTION
**Pull request by etothepi16**
Per one of my earlier pull requests, resubmitted because I went off scope on that one.
The new loop requires changing da to keep the resolution the same.

### CODE REVIEW

File: challenges/CC_60_Butterfly_Wings/sketch.js [hunk 1]
```diff
@@ -18,22 +18,12 @@ function draw() {
...
-    var r = sin(2 * a) * map(n, 0, 1, 50, 300);
-    var x = r * cos(a);
-    var y = r * sin(a);
-    xoff += dx;
-    //point(x, y);
-    vertex(x, y);
-  }
-
-  for (var a = PI / 2; a <= 3 * PI / 2; a += da) {
+  for (var a = -PI / 2; a <= 3 * PI / 2; a += da) {
```
Comment (shiffman) [inline challenges/CC_60_Butterfly_Wings/sketch.js]: I didn't test this cod but doesn't there need to be something to increment xoff for half the loop and decrement it for the other half?

Comment (etothepi16) [inline challenges/CC_60_Butterfly_Wings/sketch.js]: Yes, if you want it to have symmetry. Personally, I like the asymmetrical look. You can get symmetry back by adding `xoff %= 5;`

Comment (etothepi16) [inline challenges/CC_60_Butterfly_Wings/sketch.js]: True. fixed

**Comment by shiffman** [comment]
nice, thank you!!

**Pull request merged**